### ISSUE TITLE
More sources

### ISSUE THREAD

**Issue by ZacSweers** [ZacSweers/CatchUp#3]
Options are nice. What do people want?

**Comment by ronshapiro** [comment]
NYTimes (@digitalbuddha)?

**Comment by digitalbuddha** [comment]
I don't believe we paginate but here are all our public feeds http://www.nytimes.com/services/xml/rss/index.html

**Comment by digitalbuddha** [comment]
We also have static json we publish 2x daily 
 Homepage - https://static01.nyt.com/services/json/sectionfronts/index.json, 
Sections https://static01.nyt.com/services/json/sectionfronts/business/index.json

**Comment by hzsweers** [comment]
Is there any content difference between the rss/json options? If not I'll probably go with whichever one has the smaller payload since we won't show the full details

**Comment by Boot-Error** [comment]
Add 4chan /g/ and /sci/

**Comment by digitalbuddha** [comment]
Nope they are the same.

**Comment by TylerMcCraw** [comment]
/r/androiddev/ subreddit maybe? (even if it's read only)
Looks like their API offers pagination https://www.reddit.com/dev/api/

**Comment by hzsweers** [comment]
yeah paginating the androiddev subreddit wouldn't require any added logic over the current front page logic on the existing reddit controller. Specific subreddits support would probably come in the form of per-service preferences

**Comment by ibrahimAlii** [comment]
Enable notifications..

**Comment by jbarr21** [comment]
Lemmy and/or KBin

**Comment by JohnBuhanan** [comment]
Plus 1 for Lemmy.  Reddit recently did away with their 3rd party APIs.
